test/fixtures/wpt/README.md

```diff
@@ -21,7 +21,7 @@ Last update:
 - html/webappapis/timers: https://github.com/web-platform-tests/wpt/tree/5873f2d8f1/html/webappapis/timers
 - interfaces: https://github.com/web-platform-tests/wpt/tree/79fa4cf76e/interfaces
 - resources: https://github.com/web-platform-tests/wpt/tree/972ca5b669/resources
-- url: https://github.com/web-platform-tests/wpt/tree/1fcb39223d/url
+- url: https://github.com/web-platform-tests/wpt/tree/41590413af/url
 
 [Web Platform Tests]: https://github.com/web-platform-tests/wpt
-[`git node wpt`]: https://github.com/nodejs/node-core-utils/blob/master/docs/git-node.md#git-node-wpt
+[`git node wpt`]: https://github.com/nodejs/node-core-utils/blob/main/docs/git-node.md#git-node-wpt
```

Comment:
Should we use `HEAD` so it always point to the "correct" branch, even if it changes name in the future?
```suggestion
[`git node wpt`]: https://github.com/nodejs/node-core-utils/blob/HEAD/docs/git-node.md#git-node-wpt
```

Reply:
Ah thanks, I didn't realize it was an automatic change. Hum it's probably not worth the hassle, we can do that if the branch name ever change again – which is unlikely tbh.